### PR TITLE
Feat/331 pino ecs  rotate stream

### PR DESCRIPTION
**Pull request by ianliuwk1019**
WIP: Do Not Merge!

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
 - [api](https://fom-418.apps.silver.devops.gov.bc.ca/api)
 - [admin](https://fom-418.apps.silver.devops.gov.bc.ca/admin)
 - [public](https://fom-418.apps.silver.devops.gov.bc.ca/public)

Once merged, code will be promoted and handed off to following workflow run.
 - [Main Merge Workflow](https://github.com/bcgov/nr-fom/actions/workflows/merge-main.yml)

### CODE REVIEW

File: api/src/app/app.module.ts [hunk 1]
```diff
@@ -71,3 +61,88 @@ function getLogLevel():string {
...
+      level: getLogLevel(),
+    },
+    pino.multistream(streams)] // multistream so can have stdout & file
+  }
+
+  return logParams;
+}
+
+/*
+// Using transport with target(pion/file, pino-pretty), not ecs format.
```
Comment (basilv) [inline api/src/app/app.module.ts]: We should clean this up (remove the commented out stuff).

File: docker-compose.yml [hunk 2]
```diff
@@ -52,6 +52,7 @@ services:
       - BYPASS_CORS=true
       - OBJECT_STORAGE_URL=nrs.objectstore.gov.bc.ca
       - OBJECT_STORAGE_ACCESS_ID=nr-fom-dev
+      - LOG_ROTATE_FILESIZE=100K
```
Comment (basilv) [inline docker-compose.yml:55]: Need to think about this. I'd likely prefer a daily rotation if that's an option.

Comment (ianliuwk1019) [reply docker-compose.yml:55]: I temporarily set it to 100K for testing in Openshift. Was thinking to set it to 10M.
The reason (but not meant that I don't want) I didn't use daily was the package seem to rotate log even if log is empty (maybe that's general default behavior even for 'logrotate'). There is no option to add for not rotating if empty. But that's probably small issue.

**Comment by DerekRoberts** [comment]
Three months stale.  Closing.

**Comment by basilv** [comment]
This is left open because it is limbo and will be picked up later.

**Comment by DerekRoberts** [comment]
> This is left open because it is limbo and will be picked up later.

Okay.  I'll delete the associated resources, so it'll at least run light. @basilv

**Comment by basilv** [comment]
Sounds good, thanks!

**Pull request closed without merging**